server/src/milvus/milvus.service.ts

```diff
@@ -95,26 +101,12 @@ export class MilvusService {
         }),
       });
 
-      // Create a new database service and check if the specified database exists
-      let hasDatabase = false;
-      try {
-        hasDatabase = await this.databaseService.hasDatabase(
-          milvusClient.clientId,
-          database
-        );
-      } catch (_) {
-        // ignore error
-      }
-
-      // if database exists, use this db
-      if (hasDatabase) {
-        await this.databaseService.use(milvusClient.clientId, database);
-      }
+      await this.databaseService.use(milvusClient.clientId, database);
```

Comment:
when we use {database: test, username: test, password: 123} to connect.
Because we can not set database before this.databaseService.hasDatabase, this api will always throw error, then hasDatabase is always false.